### ISSUE TITLE
Create and load user profiles (for authoring nanopubs)

### ISSUE THREAD

**Issue by raar1** [fair-workflows/nanopub#9]
A user very likely wants to use the same key (and introductory nanopub) that they normally use. Would it then make sense to have some sort of ```NanoUserProfile``` class? Or a ```nano_user_profile.yml``` or something like that, which would contain the introductory nanopub and the path to the RSA keys they want to use?

Perhaps we then need to force a user to explicitly load the user profile they want to act as. We need not require this for searching and fetching, but if the user wants to e.g. ```publish()``` then I suppose they should need to provide an ```author=``` kwarg that gives this user info?

**Comment by svenvanderburg** [comment]
- Instead of `make_nanopub_keys`, we have something like `setup_profile`. This will do:
-- Profile creation: Create a nanopublication linking RSA key to author. Use ORCID-ID preferably.
-- Use existing RSA keys and 'this-is-me' publication.
- Store link to nanopub
- Every publication should automatically add this link

**Comment by svenvanderburg** [comment]
It is unclear how exactly the 'hello-this-is-me' nanopublication should look like. @raar1 do we have an example of this, or should we just be creative?

**Comment by raar1** [comment]
Hi @svenvanderburg , this is the intro nanopub that was generated for me by nanobench (I think): [http://server.nanopubs.lod.labs.vu.nl/RAy1CYBfBYFd_TFI8Z_jr3taf6fB9u-grqsKyLzTmMvQI](http://server.nanopubs.lod.labs.vu.nl/RAy1CYBfBYFd_TFI8Z_jr3taf6fB9u-grqsKyLzTmMvQI)

**Comment by dsmits** [comment]
Initial this-is-me nanopubs and the following ones will be linked to the author using the ORCID in the following way:
```
sub:provenance {
  sub:assertion prov:wasAttributedTo orcid:0000-0002-9984-2720 .
}
```

**Comment by raar1** [comment]
My understanding was that later nanopubs should be attributed to the intro nanopub itself (not the ORCID), just because it's possible someone doesn't have an ORCID so other identifying info will be in the intro nanopub. The public key in the intro nanopub I think is the most important identifying information (from a nanopub perspective).

**Comment by tkuhn** [comment]
The subsequent nanopubs should be signed with the public key announced in the introduction, and should be using the same URI (preferably ORCID) to identify the user in the pubinfo (as dct:creator). The provenance part can be different (you can publish something that had a different source). There is no need to point to the intro nanopub itself (only via public key + ORCID).

**Comment by raar1** [comment]
Thanks for the clarification @tkuhn. In that case the idea would be that essentially everyone making nanopublications has an ORCID, then?

**Comment by tkuhn** [comment]
It can be any URI, doesn't have to be an ORCID necessarily. But it makes things easier if we nudge users into using ORCID for it :). And there really isn't much of a reason against ORCID.

**Comment by svenvanderburg** [comment]
Fixed with #31, #32, #33. We have a very strong nudge for using ORCID.
